be/src/storage/rowset/segment_iterator.cpp

```diff
@@ -166,6 +166,10 @@ class SegmentIterator final : public ChunkIterator {
 
         // not all dict encode
         bool _has_force_dict_encode{false};
+
+        // save overflow rows in read_chunk
+        std::shared_ptr<Chunk> _overflow_read_chunk;
```

Comment:
_overflow_read_chunk seems that it has never been used, it is all used context->_overflow_read_chunk